css/properties/border-block-end-color.json

```diff
@@ -0,0 +1,50 @@
+{
+  "css": {
+    "properties": {
+      "border-block-end-color": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/border-block-end-color",
+          "support": {
+            "chrome": {
+              "version_added": false
+            },
+            "webview_android": {
+              "version_added": false
+            },
+            "firefox": {
+              "version_added": "41",
+              "notes": "Available since Firefox 38, but behind the preference <code>layout.css.vertical-text.enabled</code>, disabled by default. The preference has been removed in Firefox 51 and this property cannot be disabled since this version."
```

Comment:
We do it like this:
"firefox": [
  {
     "version_added": "41"
  },
  {
    "version_added": "38",
    "version_removed": "51",
    "flag": 
                "flag": {
                  "type": "preference",
                  "name": "layout.css.vertical-text.enabled",
                  "value_to_set": "true"
                }
  }  
]

cc/ @Elchi3 for confirmation

Reply:
Thanks Jean-Yves, this is a good call I think. I hadn't thought about this earlier, but it probably makes sense this way. Confirming that we should use proper data structure here instead of just a note.